doc/Pacemaker_Explained/en-US/Ch-Resources.txt

```diff
@@ -519,7 +519,7 @@ but moderate. The minimum timeouts should never be below 10 seconds.
 Location to store the resource state in.
 </longdesc>
 <shortdesc lang="en">State file</shortdesc>
-<content type="string" default="/var/run//Dummy-{OCF_RESOURCE_INSTANCE}.state" />
+<content type="string" default="/var/run//Dummy-default.state" />
```

Comment:
What about escaping the `$` so that it appears literally in the metadata? And maybe explaining in the description that the default depends on the (yet unknown) resource name. Is the `default` attribute used for anything by the tools?